tests/test_cli_wrapper.py

```diff
@@ -40,8 +40,20 @@ def test_request_body(self):
         tag = {'name': 'Tag'}
         name = 'some-name'
         debug_url = 'debug-url'
-        response = self.cli_wrapper._request_body(name, tag, [tile], debug_url)
+        ignored_elements_data = {
```

Comment:
add a new test instead of updating existing as this is an optional param which might be null

Reply:
request body doesn't have optional params as the function that calls have optional params

Reply:
Sure, but it validates that you are not calling a function on that arg, which can be None.

Say if your body generation included `arg.to_json()` that would break if arg was none

Reply:
yep i have written new tests for it.